src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        final Injector injector = Injector.getInstance("mate.academy");
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setDescription("");
+        cinemaHall.setCapacity(1);
+        cinemaHallService.add(cinemaHall);
+        cinemaHallService.get(1L);
```

Comment:
Id must be generated by the database. Do not hardcode it in the code
```suggestion
        cinemaHallService.get(cinemaHall.getId());
```